features/polyExplorer/src/components/clusterStories/sectionTitle.css

```diff
@@ -1,22 +1,21 @@
 .section-title-container {
-  width: 80%;
-  display: grid;
-  grid-template-columns: 2fr 1fr 2fr;
-  grid-gap: 7px;
-  align-items: center;
+    display: grid;
+    grid-template-columns: 1fr 1fr 1fr;
+    grid-gap: 12px;
```

Comment:
This is the kind of thing that should probably have some centralized place. Going from 7 to 12 px might seem unimportant... Until you have to refactor all grid gaps (whatever that is). The other quantities have been changed, I'm sure for a good reason, but still...

Reply:
I think it makes more sense to take this css with the component and add it to polyLook soon

Reply:
Yes, this is a great "candidate component" for the polyLook.